homeassistant/components/insteon_local.py

```diff
@@ -71,6 +78,11 @@ def setup(hass, config):
             _LOGGER.error("Error on insteon_local hub check", exc_info=True)
         return False
 
+    linked = insteonhub.get_linked()
+
     hass.data['insteon_local'] = insteonhub
 
+    for insteon_platform in INSTEON_PLATFORMS:
+        load_platform(hass, insteon_platform, DOMAIN, {'linked': linked})
```

Comment:
I think you should pass the original config as the fifth argument here. That will make sure the components are setup properly.